db/migrate/20200910163614_adjust_control_explorer_features.rb

```diff
@@ -0,0 +1,48 @@
+class AdjustControlExplorerFeatures < ActiveRecord::Migration[5.2]
+  class MiqProductFeature < ActiveRecord::Base; end
+  class MiqRolesFeature < ActiveRecord::Base; end
+
+  FEATURE_MAPPING = {
+    'control_explorer'  => 'control'
+  }.freeze
+
+  def up
+    return if MiqProductFeature.none?
+
+    say_with_time 'Adjusting control_explorer features for the split explorers' do
+      new_feature = MiqProductFeature.find_or_create_by!(:identifier => 'control')
+      user_role_ids_with_old_features = MiqRolesFeature.where(:miq_product_feature_id => old_feature_ids).pluck(:miq_user_role_id).uniq
+      user_role_ids_with_old_features.each do |user_role_id|
+        MiqRolesFeature.create!(:miq_product_feature_id => new_feature.id, :miq_user_role_id => user_role_id)
+      end
+
+      # Direct renaming of features
+      FEATURE_MAPPING.each do |from, to|
+        MiqProductFeature.find_by(:identifier => from)&.update!(:identifier => to)
```

Comment:
I'm not sure but, wouldn't this result in creating 2 product features where the identifier is `control`? Because on line 13 will create the new one and this line will find the old one and update the identifier to `control` as well

Reply:
@gtanzillo https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_product_feature.rb#L20 should not allow duplicates, correct?

Reply:
@h-kataria Since you stubbing that model than no code in core is being executed, so no.

Reply:
I think this does create 2 control entries...you should be able to verify by changing the `it 'renames the features' do` spec to count the number of "control" product features and ensure there is only one.